### PR TITLE
Load balancer

### PR DESCRIPTION
**Pull request by delapuente**
@marco-c do you mind to take a look at the last recipe for more than offline, please?

### CODE REVIEW

File: load-balancer/index.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  }).then(function(result) {
+    $('#loads-label').textContent = result;
+  });
+};
+
+// Simply change the source for the image.
+$('#image-selector').onchange = function() {
+  var imgUrl = $('select').value;
+  if (imgUrl) {
+    // The bumping parameter `_b` is just to avoid HTTP cache.
```
Comment (delapuente) [inline load-balancer/index.js]: This is a bug in Firefox, I'm checking with platform.

File: load-balancer/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Intermediate
+
+## Use case
+As a service provider, I want to provide the best source in terms of availability for a selected resource.
+
+## Solution
+Use a Service Worker to intercept the requests to the resources and select the proper content provider accordingly to their availability.
+
+## Category
+More than offline
```
Comment (darkwing) [inline load-balancer/README.md]: "More than Offline"

Comment (darkwing) [inline load-balancer/README.md]: Ooops!  Never mind, my PR isn't in yet.  Keep as is!

Comment (darkwing) [inline load-balancer/README.md]: Sorry, I lied, it is "More than Offline"

Comment (delapuente) [inline load-balancer/README.md]: Done, if you want to take a deeper look, here we are a little bit busy and it's the last recipe I need to publish the blog post. Thx.

Comment (marco-c) [inline load-balancer/README.md]: It should be "Beyond Offline" now

File: load-balancer/index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Load balancer - ServiceWorker Cookbook</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+  <h2>Resources</h2>
+  <p>
+    Choose from one the images to be loaded:
```
Comment (darkwing) [inline load-balancer/index.html]: Choose from one *of* ....

**Comment by darkwing** [comment]
I think we need some sort of additional feedback within the recipe to show what's happening under the hood; I see that the server load items are configurable but when I switch images it isn't apparent to me what magic is happening -- the new image displays but that's it -- I don't see where it's coming from and how it's different than the other servers.

Also, Chrome is giving me this error upon every load of the example page:

GET http://localhost:3003/load-balancer/service-worker.js net::ERR_FILE_EXISTS

**Comment by marco-c** [comment]
@darkwing the images are different (they contain the name of the server they're coming from).

Have you tried it in Firefox? Does it crash for you in Nightly?

**Comment by darkwing** [comment]
@marco-c I totally missed that :(  Yep, that's awesome.  

Nit:  When you choose an image, then choose "Select an Image" again, the image 404's; we should do something in that case.  Maybe we default to image 1?

**Comment by delapuente** [comment]
Fixing what you pointed to me.

**Comment by delapuente** [comment]
> Also, Chrome is giving me this error upon every load of the example page:

> GET http://localhost:3003/load-balancer/service-worker.js net::ERR_FILE_EXISTS

This is an known error: http://stackoverflow.com/questions/34126211/polymer-starter-kit-err-file-exists-when-trying-to-enable-service-worker#comment56032642_34126211

**Comment by delapuente** [comment]
And, for the sake of completion, here is the bug I filed about the weird crash we were experimenting these days: https://bugzilla.mozilla.org/show_bug.cgi?id=1231978

**Comment by darkwing** [comment]
R+, works for me!

**Pull request merged**